po/zh-TW.po

```diff
@@ -3588,17 +3659,19 @@ msgid ""
 "is\n"
 "raised."
 msgstr ""
+"只要在物件中包裝指標，即可在物件刪除時釋放記憶體。即使發生例外狀況，編譯器仍"
+"會保證執行這項作業。"
 
 #: src/memory-management/scope-based.md:9
 msgid ""
 "This is often called _resource acquisition is initialization_ (RAII) and "
 "gives\n"
 "you smart pointers."
-msgstr ""
+msgstr "這通常稱為「資源取得即初始化」(RAII)__，且會提供智慧指標。"
```

Comment:
```suggestion
msgstr "這通常稱為「資源取得即初始化」(RAII)，且會提供智慧指標。"
```

Could you tell the translation team to remove future markup ** and __ once they replaced them with 「」?

Reply:
> Could you tell the translation team to remove future markup ** and __ once they replaced them with 「」?

@henrif75, good advice for next iteration. As I understand it, 「」 is used for emphasis, so the Markdown characters should go afterwards.